batch/sql/add-user-resources-ready-cores-token.py

```diff
@@ -0,0 +1,17 @@
+import asyncio
+from gear import Database
+
+
+async def main():
+    db = Database()
+    await db.async_init()
+
+    await db.just_execute('CALL insert_ready_cores_tokens();')
+
+    users = db.execute_and_fetchall('SELECT user FROM user_resources;')
+    async for user in users:
+        await db.just_execute('CALL insert_user_resources_tokens(%s)', (user,))
```

Comment:
OK, I'm going to propose a slightly different, more flexible design: don't pre-insert into these tables.  When updating the stats with a token, use INSERT ... ON DUPLICATE KEY UPDATE.  This will allow you to insert or update without knowing if the key exists.

Then, I propose we put the number of tokens in the globals, and query it when generating the random value.  This will let us easily adjust the number of tokens in the future.   Let's start with a default of 200.